src/app/domain/schema-management/components/manual-verification-wizard.component.html

```
@@ -0,0 +1,122 @@
+                <th scope="col" class="px-4 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">Verify</th>
+                <th scope="col" class="px-4 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">Row</th>
+                <th scope="col" class="px-4 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">Subject ID</th>
+                <th scope="col" class="px-4 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">Treatment</th>
+                <th scope="col" class="px-4 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">Strata</th>
+              </tr>
+            </thead>
+            <tbody class="bg-white dark:bg-slate-800 divide-y divide-border-subtle">
+              @for (sample of samples(); track sample.rowIndex; let i = $index) {
+                <tr [class.bg-green-50]="sample.verified" [class.dark:bg-green-900/20]="sample.verified" class="transition-colors">
+                  <td class="px-4 py-3 whitespace-nowrap">
+                    <input type="checkbox" [checked]="sample.verified" (change)="toggleVerification(i)"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)